spec/pubid_ieee/identifier_spec.rb

```diff
@@ -480,8 +480,22 @@ def context_description(ex)
     it_behaves_like "converts pubid to pubid"
   end
 
-  context "IEEE Std 802.11ae-2012" do
-    let(:pubid) { "IEEE Std 802.11ae-2012" }
+  context "IEEE Std 1003.1-2001/Cor 2-2004" do
+    let(:pubid) { "IEEE Std 1003.1-2001/Cor 2-2004" }
```

Comment:
Why does this have a space between `Cor 2` but the others are like `Cor1`?

Reply:
> Why does this have a space between `Cor 2` but the others are like `Cor1`?

There are many ways of how they can be represented:
```
IEEE Std 802.1Q-2005/Cor1-2008 (Corrigendum to IEEE Std 802.1Q-2005)
IEEE Std C62.41.2-2002_Cor 1-2012 (Corrigendum to IEEE Std C62.41.2-2002)
IEEE PC57.12.90-2015/Cor1/D3.1, September 2016
IEEE P802.1AB-Cor-1/D2.0, January 2013
IEEE Std 535-2013/Cor. 1-2017 (Corrigenda of IEEE Std 535-2013)

```

Reply:
I found these patterns:

```
grep -e 'Cor\.' pubid-to-parse.txt | wc -l
      10
grep -e 'Cor\d' pubid-to-parse.txt | wc -l
      90
grep -e 'Cor \d' pubid-to-parse.txt | wc -l
     131
```

Out of them the last one is most popular.

Let's use `Cor \d` (and `Cor \d-yyyy`) as the pattern.